.env

```diff
@@ -6,7 +6,7 @@ RESTART_POLICY=no
 TAG=latest
 QUERY_PORT=127.0.0.1:5511
 POSTGRES_IMAGE=pgautoupgrade/pgautoupgrade:16-alpine
-POSTGRES_HOST=localhost
+POSTGRES_HOST=localhost:5512
```

Comment:
the variables POSTGRES_HOST and POSTGRES_PORT are a bit confusing 
both contain host + port.
maybe we could have some comments to explain what they are, why they are different etc.